components/automate-cs-nginx/habitat/plan.sh

```diff
@@ -18,7 +18,7 @@ pkg_deps=(
   # chef-server-* packages are pinned to the versions required by the
   # chef-server-* packages.
   #
-  core/curl
+  core/curl/7.79.1/20240105235730
   core/ruby27
   # WARNING: Version pin managed by .expeditor/update_chef_server.sh
   "${vendor_origin}/chef-server-nginx/15.4.0/20240129214124"
```

Comment:
Use the latest version. Need to rebase when the Infra Server packages are merged to main